rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/custom_service_info.hpp

```diff
@@ -82,11 +82,7 @@ class ServicePubListener : public eprosima::fastrtps::PublisherListener
       rmw_fastrtps_shared_cpp::hash_fastrtps_guid>;
 
 public:
-  explicit ServicePubListener(CustomServiceInfo * info)
-  : info_(info)
```

Comment:
We can also remove the `info_` member.

Reply:
Done. This should also fix both Clang warnings in macOS.